app/allocations/middleware/set-results.allocations.js

```diff
@@ -3,17 +3,27 @@ const allocationService = require('../../../common/services/allocation')
 
 async function setResultsAllocations(req, res, next) {
   try {
-    const allocations = await allocationService.getByDateAndLocation(
-      req.body.allocations
-    )
+    const [activeAllocations, cancelledAllocations] = (
+      await Promise.all([
+        allocationService.getByDateAndLocation({
```

Comment:
Can we extract this to the service level. Any hardcoded statuses that relate to the API should live in the service so that we contain that logic to one place.

This file should make one call to something like `allocationService.getActive(req.body.allocations)` and `allocationService.getCancelled(req.body.allocations)`.

This layer should be super simple and just call the relevant service method.